helix-core/src/test/java/org/apache/helix/task/TestTaskStateModelFactory.java

```diff
@@ -95,6 +95,7 @@ public void testZkClientCreationMultiZk() throws Exception {
         .getTargetThreadPoolSize(zkClient, anyParticipantManager.getClusterName(),
             anyParticipantManager.getInstanceName()), TEST_TARGET_TASK_THREAD_POOL_SIZE);
     Assert.assertTrue(zkClient instanceof FederatedZkClient);
+    zkClient.close();
```

Comment:
Do you think add thins `zkClient.close()` to a final block may help? If assert fail then this Utest fails, we will miss the close.

Reply:
Very true, let me fix it up.